intyg-proxy-service/app/src/main/java/se/inera/intyg/intygproxyservice/citizen/service/Elva77ResponseConverter.java

```diff
@@ -0,0 +1,34 @@
+package se.inera.intyg.intygproxyservice.citizen.service;
+
+import org.springframework.stereotype.Component;
+import se.inera.intyg.intygproxyservice.citizen.dto.CitizenDTO;
+import se.inera.intyg.intygproxyservice.citizen.dto.CitizenResponse;
+import se.inera.intyg.intygproxyservice.integration.api.elva77.Elva77Response;
+import se.inera.intyg.intygproxyservice.integration.api.pu.Status;
+
+@Component
+public class Elva77ResponseConverter {
```

Comment:
Do we need a separate converter for this or can we place it in the object class?